src/App.js

```diff
@@ -26,8 +27,25 @@ export class App extends Component {
     }
   }
 
+  async initTelemetry() {
+    // embed checks if explorer is running stand alone or embeded in webui
+    const { embed } = this.props.queryObject
+    const { telemetry } = this.state
+
+    if (telemetry == null && !embed) {
+      const { BrowserMetricsProvider } = await import('@ipfs-shipyard/ignite-metrics/dist/src/BrowserMetricsProvider')
+      const newTelemetry = new BrowserMetricsProvider({ appKey: '8bf7ee9fa77ec75c8173aa9fdc4877f5e2b61628' })
+      window.telemetry = newTelemetry
+      window.removeMetricsConsent = () => newTelemetry.removeConsent(['minimal'])
+      window.addMetricsConsent = () => newTelemetry.addConsent(['minimal'])
+
+      this.setState({ telemetry: newTelemetry })
+    }
+  }
+
   componentDidMount() {
     this.props.doInitIpfs()
+    this.initTelemetry()
```

Comment:
i'd actually swap these
```suggestion
    this.initTelemetry()
    this.props.doInitIpfs()
```